src/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,56 @@
+import { FilePlus, FolderPlus, Home, Share2, Files } from "lucide-react";
+
+export function Sidebar() {
+  return (
+    <div className="col-span-2 row-span-5 h-full bg-gray-200">
+      <div className="flex flex-col justify-between">
+        <div>
+          <div className="mb-2 ml-1 mr-1 mt-4">
+            <button
+              className="w-full rounded-full bg-blue-600 px-4 py-2 text-white hover:bg-blue-700"
+              aria-label="Upload files"
+            >
+              <div className="flex items-center justify-center">
```

Comment:
According to [this](https://caninclude.glitch.me/caninclude?child=div&parent=button), it's not correct to include a `div` inside a `button`. Also, it seems that the div isn't needed, you can just apply the classes directly to the `button`.